### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by TimOsmond**
Hi @mattglenjackson, could you have a look over this... Thanks :)

### CODE REVIEW

File: prac_06/guitars.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+"""
+Store a list of guitars and print details.
+"""
+from prac_06.guitar import Guitar
```
Comment (mattglenjackson) [inline prac_06/guitars.py:4]: I've noticed you don't have your guitar.py for your guitar class in your repo. Make sure you put that in before Wednesday :)

File: prac_06/guitar_test.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+"""
+
+from prac_06.guitars import Guitar
+
+
+def main():
+    name1 = "Gibson L-5 CES"
+    year1 = 1922
+    cost1 = 16035.40
+    guitar = Guitar(name1, year1, cost1)
```
Comment (mattglenjackson) [inline prac_06/guitar_test.py:13]: Nothing really wrong with this, but you showed an easier way to do this on the line below that avoids multiple variables assignments.

**Review by mattglenjackson — COMMENTED** [review]
Only really one minor thing that could be improved. Good job this week Tim. Remember to push your guitar.py for your guitar class before the due date!

File: prac_06/guitars.py (hunk 1 above)
Comment (TimOsmond) [reply prac_06/guitars.py:4]: Well spotted - thanks Matt :)

File: prac_06/guitar_test.py (hunk 2 above)
Comment (TimOsmond) [reply prac_06/guitar_test.py:13]: Thanks Matt, good idea...

**Pull request merged**